reconcile/utils/terrascript_aws_client.py

```diff
@@ -1391,15 +1399,29 @@ def populate_tf_resource_rds(self, spec):
 
         if len(deps) > 0:
             values["depends_on"] = deps
-
+        
         # pop alternate db name from value before creating the db instance
         # this will only affect the output Secret
         output_resource_db_name = values.pop("output_resource_db_name", None)
 
+        if "event_notifications" in values.keys():
+            event_notifications = values.pop("event_notifications") 
+            counter = 0
+            for e_n in event_notifications:
+                sns_topic_name = e_n.pop("destination")
+                e_n["sns_topic"] = "${aws_sns_topic"+ "." + sns_topic_name + ".arn}"
+                e_n["source_ids"] = ["${aws_db_instance"+ "." + identifier + ".id}"]
+                print(e_n["source_ids"])
+                counter += 1
+                e_n_identifier = f"{identifier}_aws_db_event_subscription_{counter}"
```

Comment:
Does this mean that if we removed say the first subscription that every other subscription would be recreated?

Reply:
Actually I realize that the primary key to the resource can be the SNS topic name combined with `source_type` in this case. Tenant shouldn't be creating separate event notifications for the same `source_type` but different `event_categories`.